src/js/controller/settings/exportimage/ImageExportController.js

```diff
@@ -17,6 +17,11 @@
     this.addEventListener(scalingFactorInput, 'input', this.onScalingFactorChange_);
     this.updateScalingFactorText_(scalingFactorInput.value);
 
+    var framesPerRowsInput = document.querySelector('.export-frames-per-rows-field');
+    framesPerRowsInput.value = pskl.UserSettings.get(pskl.UserSettings.FRAMES_PER_ROWS);
+    this.addEventListener(framesPerRowsInput, 'change', this.onFramesPerRowsChange_);
+    this.addEventListener(framesPerRowsInput, 'input', this.onFramesPerRowsChange_);
+
```

Comment:
Just what you are adding for this feature, thanks!
I will file a cleanup issue for this controller afterwards.